src/extension.ts

```diff
@@ -34,6 +35,9 @@ export function activate(context: vscode.ExtensionContext) {
   context.subscriptions.push(generateCodeSnippetDisposable);
   console.log(">>> generateCodeSnippetDisposable is now active!");
 
+  // Automatically check L2 version on extension activation
+  checkL2Version();
```

Comment:
So if version is too old, what happens to the variable  suggestion? Is it totally dead in older versions?

Reply:
No in the older version what we were doing is searching the le.env file in front end and suggesting it. So it won't be a problem.

Reply:
What I am saying is something else.

Say, the extension frontend updates, but the l2 version is still old, what happens?